packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m

```diff
@@ -584,7 +587,10 @@ - (void)updateAutoMediaPlaybackPolicy:(NSNumber *)policy
       if (@available(iOS 10.0, *)) {
         configuration.mediaTypesRequiringUserActionForPlayback = WKAudiovisualMediaTypeNone;
       } else if (@available(iOS 9.0, *)) {
```

Comment:
This was missed during a previous cleanup round apparently; we only support iOS 9 now, so this check isn't necessary, nor is the `else` block below. This should just become `else`, and the last block removed entirely.

Reply:
Hello Stuart, am I getting this right, that with only supporting iOS 9 now you mean, that iOS 9 is the lowest supported version? I make the appropriate changes.

Reply:
fixed.

Reply:
Correct, sorry for the bad wording there.